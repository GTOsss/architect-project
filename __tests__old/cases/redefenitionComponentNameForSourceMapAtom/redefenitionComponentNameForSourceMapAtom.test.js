const appRoot = process.cwd();
const { resolve } = require('path');
const {
  returnOutputPathsWithContent,
  promisifyCliCommand,
  SnapshotSerializer,
} = require('../../utils/utilsForArcStart');
const cleanOutPutBeforeTest = require('../../utils/cleanOutPutBeforeTest');
const { pathForCommand, casesOutputPath } = require('../casesConfigPath');

const currentCommand = pathForCommand.redefenitionComponentNameForSourceMapAtom;

describe('redefenition Component Name For SourceMapAtom', () => {
  cleanOutPutBeforeTest(casesOutputPath.redefenitionComponentNameForSourceMapAtom);

  test('redefenition Component Name For SourceMapAtom', async () => {
    await promisifyCliCommand(`arc s -c ${currentCommand}`);
    const currentPathToOutput = resolve(appRoot, currentCommand, 'output');

    SnapshotSerializer();

    expect(returnOutputPathsWithContent(currentPathToOutput)).toMatchSnapshot();
  });
});
