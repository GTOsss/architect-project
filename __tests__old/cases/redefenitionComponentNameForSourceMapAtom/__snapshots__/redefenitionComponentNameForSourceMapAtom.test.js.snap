// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`redefenition Component Name For SourceMapAtom redefenition Component Name For SourceMapAtom 1`] = `
Object {
  "src": Array [
    Object {
      "content": "",
      "path": "*/redefenitionComponentNameForSourceMapAtom/output/src/page/custom/custom.module.scss",
    },
    Object {
      "content": "import React from 'react';
import { storiesOf } from '@storybook/react';
import centered from '@storybook/addon-centered/react';
import Custom from './custom';

const stories = storiesOf('custom', module).addDecorator(centered);

stories.add('default', () => {

  return <Custom />;
});
",
      "path": "*/redefenitionComponentNameForSourceMapAtom/output/src/page/custom/custom.stories.tsx",
    },
    Object {
      "content": "import React from 'react';
import styles from './Custom.module.scss';

type Props = {};

export const Custom = ({}: Props) => {
  return (
    <div>
      content
    </div>
  );
};

export default Custom;
",
      "path": "*/redefenitionComponentNameForSourceMapAtom/output/src/page/custom/custom.tsx",
    },
    Object {
      "content": "import Custom from './custom';

export default Custom;
",
      "path": "*/redefenitionComponentNameForSourceMapAtom/output/src/page/custom/index.ts",
    },
    Object {
      "content": "import Stadiums from './stadiums';

export default Stadiums;
",
      "path": "*/redefenitionComponentNameForSourceMapAtom/output/src/page/stadiums/index.ts",
    },
    Object {
      "content": "",
      "path": "*/redefenitionComponentNameForSourceMapAtom/output/src/page/stadiums/stadiums.module.scss",
    },
    Object {
      "content": "import React from 'react';
import { storiesOf } from '@storybook/react';
import centered from '@storybook/addon-centered/react';
import Stadiums from './stadiums';

const stories = storiesOf('stadiums', module).addDecorator(centered);

stories.add('default', () => {

  return <Stadiums />;
});
",
      "path": "*/redefenitionComponentNameForSourceMapAtom/output/src/page/stadiums/stadiums.stories.tsx",
    },
    Object {
      "content": "import React from 'react';
import styles from './Stadiums.module.scss';

type Props = {};

export const Stadiums = ({}: Props) => {
  return (
    <div>
      content
    </div>
  );
};

export default Stadiums;
",
      "path": "*/redefenitionComponentNameForSourceMapAtom/output/src/page/stadiums/stadiums.tsx",
    },
  ],
}
`;
